src/components/PlatformNavigator/__tests__/PlatformNavigator.test.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { PlatformNavigator } from '../index';
+import userEvent from '@testing-library/user-event';
+
+const routerMock = {
+  __esModule: true,
+  useRouter: () => {
+    return {
+      pathname: ''
+    };
+  }
+};
+
+jest.mock('next/router', () => routerMock);
+
+describe('PlatformNavigator', () => {
+  const component = (
+    <PlatformNavigator currentPlatform={'react'} isPrev={true} />
+  );
+
+  it('should render the PlatformNavigator component', async () => {
+    render(component);
+
+    const navigator = await screen.findByText('Choose your framework/language');
+    expect(navigator).toBeInTheDocument();
+  });
+
+  it('should show the selected platform as React', async () => {
```

Comment:
I think you mean default platform?
```suggestion
  it('should show the default platform as React', async () => {
```